packages/fxa-settings/src/images/index.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { ReactComponent as HeartsBroken } from './graphic_hearts_broken.svg';
+import { ReactComponent as HeartsVerified } from './graphic_hearts_verified.svg';
+import { ReactComponent as RecoveryCodes } from './graphic_recovery_codes.svg';
+import { ReactComponent as TwoFactorAuth } from './graphic_two_factor_auth.svg';
+import { ReactComponent as Mail } from './graphic_mail.svg';
+
+import { useFtlMsgResolver } from '../models';
+
+// Use this component to add your image into the collection of images exported below.
+// If the component is not being reused across directories, it should remain inline
+// If you are introducing an image which was previously shared and had localized alt-text,
+// or a localized aria-label, please add the existing ftl id to spare translating it again.
+// in the directory where it is used in accordance with the current policy (https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/README.md#inlined-svgs)
+// Don't forget to add new FTL strings into the `en.ftl` file
+
+interface PreparedImageBaseProps {
+  className?: string;
+  Image: React.ElementType;
+}
+
+interface PreparedImageAriaHiddenProps extends PreparedImageBaseProps {
+  ariaHidden: true;
+}
+
+interface PreparedImageAriaVisibleProps extends PreparedImageBaseProps {
+  ariaHidden?: false;
+  ariaLabel: string;
+  ariaLabelFtlId: string;
+}
+
+type PreparedImageProps =
+  | PreparedImageAriaHiddenProps
+  | PreparedImageAriaVisibleProps;
+
+const PreparedImage = (props: PreparedImageProps) => {
+  const { className, ariaHidden, Image } = props;
+  const ftlMsgResolver = useFtlMsgResolver();
+  const showAriaLabel =
+    !ariaHidden && props?.ariaLabel && props?.ariaLabelFtlId;
+  const localizedAriaLabel =
+    showAriaLabel &&
+    ftlMsgResolver.getMsg(props.ariaLabelFtlId, props.ariaLabel);
+
+  return (
+    <>
+      {showAriaLabel ? (
+        <Image role="img" aria-label={localizedAriaLabel} {...{ className }} />
```

Comment:
If the `localizedAriaLabel` is applied directly to the attribute with `ftlMsgResolver`, it might be necessary to modify the ftl strings so that it doesn't have that `.aria-label` on a second line. Right now they are set up for the `FtlMsg` wrapper. (I think?)

Reply:
ohhhhh I can remove that!

Reply:
I'm thinking now that if the strings are edited, I think it will trigger a request to re-localize (even if we just remove the attribute). @bcolsson - is that right?

An alternative that wouldn't impact our localizers would be to wrap the `<Image>` components with `<FtlMsg>`.

Reply:
If you add or remove an attribute from an FTL string (like `.aria-label`) you will need to update the string ID. Otherwise the source language will be updated silently and localizations will no longer match.

Reply:
Ohhhh I see -- so we don't want to change the string, but we DO want change the way we're applying the localization (moving it into a wrapping component instead of using the resolver)

Reply:
I think that's the way to go here. I'm not sure what happens if we leave it as it is (using the resolver with `.aria-label` in the ftl file), but I think that's a pattern mismatch that might prevent the aria-labels from being properly localized.

Unfortunately, our l10n tests don't yet support strings with attributes, so you'll see some test failures if `testAllL10n` is active in the test files - we'll likely need to comment those out for now until we make those test improvements in https://mozilla-hub.atlassian.net/browse/FXA-6461

Reply:
Extra note: the l10n tests previously passed for the inline images because `testL10n`/`testAllL10n` don't (yet) test l10n applied via `FtlMsgResolver`. (e.g., artificially adding errors to the l10n won't show up as an error)